src/ResourceManagement/Compute/Domain/GalleryImageVersion/Definition/IDefinition.cs

```diff
@@ -170,13 +171,15 @@ public interface IWithSource  :
         /// </summary>
         /// <param name="customImageId">The ARM id of the custom image.</param>
         /// <return>The next definition stage.</return>
+        [Obsolete("WithSourceCustomImage is deprecated, source image was removed from REST API")]
```

Comment:
Do we have alternative methods for this API?

Reply:
I find it may be move into `GalleryImageVersionStorageProfile`, I would change this implementation

Reply:
fix, it would behave same as previous